docs/setup_instructions/mac_installation_instructions.md

```diff
@@ -11,7 +11,7 @@ title: macOS Installation Instructions
 
 ## Apple Command Line Tools
```

Comment:
Just noting from another PR that we sometimes call these "Developer tools" but Command Line Tools is better so we should try to be consistent